SOPT-iOS/Projects/Features/HomeFeature/Sources/HomeScene/ViewModel/HomeForMemberViewModel.swift

```diff
@@ -165,6 +166,14 @@ extension HomeForMemberViewModel {
                 owner.useCase.getRecentSchedule()
             }.store(in: cancelBag)
         
+        input.cellTapped
+            .filter{ $0.section == 1 }
+            .withUnretained(self)
+            .sink(receiveValue: { owner, indexPath in
+                self.onDashBoardCellTapped?()
+            })
+            .store(in: cancelBag)
+        
```

Comment:
filter 활용 좋아요 :)
sink 클로저 내부에서 강한참조 하고 있어요!
```suggestion
        input.cellTapped
            .filter{ $0.section == 1 }
            .withUnretained(self)
            .sink(receiveValue: { owner, indexPath in
                owner.onDashBoardCellTapped?()
            })
            .store(in: cancelBag)
        
```

Reply:
옿 ㅋㅋ 매의 눈 ~ 인정합니다

Reply:
허거덩 ~~ 이런 일이 . . 수정 완료했습니다!_!